src/features/positions/screens/PositionSheet.tsx

```diff
@@ -96,13 +95,7 @@ export const PositionSheet: React.FC = () => {
   return (
     <BackgroundProvider color="surfaceSecondary">
       {({ backgroundColor }) => (
-        <SlackSheet
-          backgroundColor={backgroundColor}
-          height={IS_IOS ? '100%' : undefined}
-          additionalTopPadding={IS_IOS ? undefined : true}
-          contentHeight={IS_IOS ? undefined : getPositionSheetHeight({ position })}
```

Comment:
One thing I just realized we might not have tested is what if the position sheet is small? We might need to keep the contentHeight set to getPositionSheetHeight, but add a maxHeight.